docs/FAQ.rst

```diff
@@ -0,0 +1,286 @@
+A: In the case of CPU execution, DALI also uses multiple worker threads.
+Using DALI should produce a better performance in most cases, even for one GPU.
+Of course, the details can depend on the particular CPU and GPU and the pipeline itself,
+as well as the current GPU utilization before introducing DALI. You can check
+`our GTC22 talk <https://www.nvidia.com/gtc/session-catalog/#/session/1636559250287001p4DG>`_
+or `the blog post <https://developer.nvidia.com/blog/case-study-resnet50-dali>`_ to see this in practice.
+
+Q: Will labels, for example, bounding boxes, be adapted automatically when transforming the image data? For example when rotating/cropping, etc. If so how?
+###########################################################################################################################################################
+A: The meta-data, like bounding boxes or coordinates, will not be adapted automatically with
+the data but DALI has a set of operators, e.g.
+`bbox_paste <../supported_ops.html#nvidia.dali.fn.bbox_paste>`_,
+`random_bbox_crop <../supported_ops.html#nvidia.dali.fn.random_bbox_crop>`_ for bounding boxes or
+`coord_transform <../supported_ops.html#nvidia.dali.fn.coord_transform>`_ for sets of coordinates.
+You can find an example `here <../examples/use_cases/detection_pipeline.html>`_.
+
+Q: How easy is it, to implement custom processing steps? In the past, I had issues with calculating 3D Gaussian distributions on the CPU. Would this be possible using a custom DALI function?
+################################################################################################################################################################################################
+A: There are several ways to do it. You can write custom operators in C++/CUDA, or run arbitrary
+Python code via the Python function and Numba operators. You can learn more about this topic
+`here <../examples/custom_operations/index.html>`_.
+
+Q: Is DALI available in Jetson platforms such as the Xavier AGX or Orin?
+########################################################################
+A: At the moment we are not releasing binaries for Jetson, but it should be possible to build
+DALI from source. You can learn more about the exact steps
+`here <../compilation.html#cross-compiling-for-aarch64-jetson-linux-docker>`_.
+
+Q: Does DALI also provide a significant improvement on inference tasks regarding latency?
+#########################################################################################
+A: It depends on what base implementation we compare to, but generally, DALI gives
+the most benefit to the throughput of the training/inference because of the batch processing
+that can utilize massive parallelism of the GPUs. Still, the GPU implementations of DALI operators
+are optimized and fast, so it might reduce the inference latency.
```

Comment:
A little bit above there is a similar question:
`Is Triton + DALI still significantly better than preprocessing on CPU, when minimum latency i.e. batch-size = 1 is desired?`
How about merging them into one, or just skipping one of them?

Reply:
Merged